omod/pom.xml

```diff
@@ -171,7 +171,7 @@
 							<includeGroupIds>${project.parent.groupId}</includeGroupIds>
 							<includeArtifactIds>${project.parent.artifactId}-api</includeArtifactIds>
 							<excludeTransitive>true</excludeTransitive>
-							<includes>**/*</includes>
+							<includes>moduleApplicationContext.xml,liquibase.xml</includes>
```

Comment:
@mogoodrich - this is the area in question.  The change I made basically says - we want to copy these specific files into the omod from the api project, whereas before it was casting a wide net and somehow also picking up all of the test classes and resources (maybe the "includeArtifactIds" includes anything that starts with the id in question, so it would also pick up artifact pihcore-api-tests ?

Reply:
Strange, nice catch... I see this is in Core Apps as well and dates back to at least 2013.

do we have an idea of what this is supposed to do?  The id says "Expand moduleApplicationContext and messages"?

Assumedly we need  Talk post about this? (Maybe you've already done so, I'm behind)

Reply:
Actually, if I'm understanding the problem right, I'm only seeing this manifest itself in the PIH Code module, but maybe I'm misunderstanding?  Let's talk after standup?

Reply:
Let's take the bigger picture question of where we might need to fix this in the OpenMRS ecosystem separately from this specific change for this specific PR, and not hold this up over the bigger picture question.

Reply:
Sure, the issue now is that I'm only seeing this problem manifest itself in PIH Core, and this code is used everywhere, so I'm wondering if something else is wrong with PIH Core?  (but again, I could be misunderstanding the issue)